core/src/main/java/org/apache/iceberg/actions/BaseCommitService.java

```diff
@@ -154,7 +171,7 @@ public void close() {
       // the commit pool to finish doing its commits to Iceberg State. In the case of partial
       // progress this should have been occurring simultaneously with rewrites, if not there should
       // be only a single commit operation.
-      if (!committerService.awaitTermination(120, TimeUnit.MINUTES)) {
+      if (!committerService.awaitTermination(timeoutInMS, TimeUnit.MILLISECONDS)) {
```

Comment:
Can we also update warning with timeoutInMS? In case the timeoutInMS is changed to other customized num.

Reply:
Updated the warning.